<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowAchat</class>
 <widget class="QMainWindow" name="MainWindowAchat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>911</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Ajouter Commande</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-50</y>
        <width>641</width>
        <height>441</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>ajouter commande</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>501</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rockwell Extra Bold</family>
          </font>
         </property>
         <property name="text">
          <string>Numéro de commande</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rockwell Extra Bold</family>
          </font>
         </property>
         <property name="text">
          <string>Prix</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rockwell Extra Bold</family>
          </font>
         </property>
         <property name="text">
          <string>Fournisseur</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prix">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>100</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_numcommande">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rockwell Extra Bold</family>
          </font>
         </property>
         <property name="text">
          <string>Quantité</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_25">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>200</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rockwell Extra Bold</family>
          </font>
         </property>
         <property name="text">
          <string>Date d'achat</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_26">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rockwell Extra Bold</family>
          </font>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajoutercommande">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>300</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_quantite">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>70</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rockwell Extra Bold</family>
          </font>
         </property>
         <property name="text">
          <string>Produit</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_description">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>130</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="le_dateachat">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>200</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_produitcommande">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>240</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Gel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Javel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Savon</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="combo_fournisseur_2">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>170</y>
           <width>73</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_14">
       <attribute name="title">
        <string>Afficher commande</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>491</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_27">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Produit</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_28">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prix</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_29">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Fournisseur</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_21">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>100</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_22">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_23">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>40</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_30">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Quantité</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_31">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>160</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>130</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>130</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_32">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>220</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_24">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>220</y>
           <width>161</width>
           <height>81</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>170</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>210</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Rechercher</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Afficher Commande</string>
     </attribute>
     <widget class="QTableView" name="tabachat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>451</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>331</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>321</width>
         <height>341</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_62">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
         </font>
        </property>
        <property name="text">
         <string>Numéro de commande</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_63">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
         </font>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_64">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
         </font>
        </property>
        <property name="text">
         <string>Fournisseur</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prix_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_numcommande_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_65">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
         </font>
        </property>
        <property name="text">
         <string>Quantité</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_66">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
         </font>
        </property>
        <property name="text">
         <string>Date d'achat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_67">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
         </font>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajoutercommande_3">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>160</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_quantite_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_dateachat_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>190</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_68">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
         </font>
        </property>
        <property name="text">
         <string>Produit</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_produitachat_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>230</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_description_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_5">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>270</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_fournisseur">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>160</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>360</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>num_commande</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>quantite</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>description</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fournisseur</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>date_achat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>produit</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_supprimer_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_recherche_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>360</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QComboBox" name="colone_tri">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>440</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NUM_COMMANDE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QUANTITE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PRIX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DESCRIPTION</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FOURNISSEUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DATE_ACHAT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PRODUIT</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="ordre_tri">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>440</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ASC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DESC</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="tri">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>tri</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>430</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
